tensorly/decomposition/candecomp_parafac.py

```diff
@@ -15,8 +15,9 @@
 
 # License: BSD 3 clause
 
-def initialize_factors(tensor, rank, init='svd', svd='numpy_svd', random_state=None, 
-                       non_negative=False, normalize_factors=False):
+def initialize_factors(tensor, rank, init='svd', svd='numpy_svd', random_state=None,
```

Comment:
I chose to keep the parafac function as similar as possible with the original one, by changin instead the initialization function. Also this way, we can propagate the changes easily to other algorithms in the same file, and also to other problems such as Tucker which use a similar initialization function.